documentation/modules/configuring/ref-storage-ephemeral.adoc

```diff
@@ -5,14 +5,26 @@
 [id='ref-ephemeral-storage-{context}']
 = Ephemeral storage
 
+[role="_abstract"]
+Ephemeral data storage is transient. 
+All pods on a node share a local ephemeral storage space. 
+Data is retained for as long as the pod that uses it is running.
+The data is lost when a pod restarts. 
+Although a pod can recover data in a highly available environment.  
+
+Because of its transient nature, ephemeral storage is not recommended for production.
```

Comment:
Should we instead of `not recommended for production` say something like `should be used only for development and/or CIs`